src/main/java/com/alkemy/ong/service/impl/CategoryServiceImpl.java

```diff
@@ -35,7 +40,9 @@ public List<Category> getAllCategories() {
 
     @Override
     public void deleteCategory(String id) {
-
+ 	String queryDeleted = "UPDATE categories SET deleted=1 WHERE Id = '" + id + "'";
```

Comment:
Este codigo esta sujeto a SQL INJECTION. Revisar: [Native sql query- SQL Injection Attack](https://stackoverflow.com/questions/25640234/native-sql-query-sql-injection-attack), para nativeQuery siempre hay que utilizar parametros. Pero de todas formas, con `SQLDelete` en la entidad no es necesario utilizar NativeQuery. Podrias revisar [How to Implement a Soft Delete with Spring JPA](https://www.baeldung.com/spring-jpa-soft-delete)

Reply:
ok,lo revisaré

Reply:
No hay problema. Perdon.

Reply:
Código corregido! @sailenicolas No pidas perdón por ayudarme! Por mi parte no dudes nunca en corregirme ya que prefiero equivocarme aquí que estamos para aprender y no en el mundo laboral luego. Muchas gracias por las guias que me pasaste y por tomarte el tiempo en buscarlas.